_scripts/app1-5.R

```diff
@@ -0,0 +1,22 @@
+{
+  
+  rm(list=ls()); 
+  options(show.error.locations = TRUE);
+  
+  # Prompt user to input temperature measurement location.
+ 
+  tempLocation <- readline("What is the location of your temperature measure?: "); # Enter measurement location (E.g. Yellowknife).
```

Comment:
A more artistic issue here:
You need to scroll to read your comment, which makes it difficult if the code is long and somebody has to peruse through it -- imagine having to constantly scroll horizontally and vertically to read code.  That is why I say to keep everything within 80 characters.  But that is more of a guideline than an absolute rule.

Reply:
@belinskyc 

I wasn't sure if the character rule included comment lines or not; 

One thing I've noticed using version control is that the comment section (e.g. on this page) you are only able to put ~ 40 characters because the horizontal scroll bar is enabled.